services/movie_session.py

```diff
@@ -0,0 +1,50 @@
+from django.db.models import QuerySet
+
+from datetime import datetime, date
+
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: date,
+        movie_id: int,
+        cinema_hall_id: int,
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id,
+    )
+
+
+def get_movies_sessions(
+    session_date: str = None
+) -> QuerySet[MovieSession]:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        date_obj = datetime.strptime(session_date, "%Y-%m-%d")
+        return queryset.filter(session_date=date_obj)

```

Comment:
The filter should be `show_time__date=date_obj` instead of `session_date=date_obj` to correctly filter by date. This aligns with the `show_time` field being a `DateTimeField`.